src/components/KBreadcrumbs/KBreadcrumbs.vue

```diff
@@ -4,54 +4,35 @@
     v-bind="$attrs"
   >
     <li
-      v-for="item in items"
+      v-for="(item, idx) in items"
       :key="item.key || item.text"
       class="k-breadcrumbs-item truncate"
     >
-      <router-link
-        v-if="typeof item.to === 'object'"
-        :class="{ 'no-underline': !item.text }"
-        :title="item.title"
-        :to="item.to"
+      <component
+        :is="getComponentAttrs(item).type"
+        v-bind="getComponentAttrs(item).attrs"
+        class="no-underline"
       >
         <KIcon
           v-if="item.icon"
           :class="['k-breadcrumb-icon', { 'has-no-text': !item.text }]"
-          color="var(--grey-500)"
+          :color="KUI_COLOR_TEXT_DECORATIVE"
```

Comment:
question: don't we need to allow customization here?

Reply:
We didn't support it before, so I assumed no?

Reply:
by having a CSS variable there in the first place, that allowed for customization